### PR TITLE
Add ERC: Count Contract

### PR DESCRIPTION
**Pull request by Yogesh8287**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: ERCS/erc-7222.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
```
Comment (abcoathup) [inline ERCS/erc-7222.md:1]:
```suggestion
---
eip: 7608
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

File: ERCS/erc-7222.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+title: Count Contract
+description: Designed to manage and keep track of a numerical count.
+author: Yogesh Sahu (@yogesh8287)
+discussions-to: https://ethereum-magicians.org/t/eip-social-recovery-interface/14494
```
Comment (abcoathup) [inline ERCS/erc-7222.md:5]:
```suggestion
discussions-to: [to be created]
```
Please create a discussions topic on https://ethereum-magicians.org and add the link here.

File: ERCS/erc-7222.md (hunk 1 above)
Comment (Yogesh8287) [reply ERCS/erc-7222.md]: OK

File: ERCS/erc-7608.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
+---
+eip: 7608
+title: Count Contract
+description: Designed to manage and keep track of a numerical count.
+discussions-to: [to be created]
```
Comment (abcoathup) [inline ERCS/erc-7608.md:5]: You need to add a discussions topic to https://ethereum-magicians.org/ with a link to this PR.

File: ERCS/erc-7608.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+## Motivation
+
+Count contracts are simple, yet widely used components in various decentralized applications. Having a standard interface will make it easier for developers to integrate and interact with different count contracts, fostering a more modular and interoperable Ethereum ecosystem.
+
+## Specification
+
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+interface CountStandard {
```
Comment (abcoathup) [inline ERCS/erc-7608.md:25]: If you were to standardize around an interface for a Counters contract then you may want to review OpenZeppelin Contracts implementation: https://github.com/OpenZeppelin/openzeppelin-contracts/blob/release-v4.9/contracts/utils/Counters.sol

Please note OpenZeppelin removed Counters in v5: https://blog.openzeppelin.com/introducing-openzeppelin-contracts-5.0

**Pull request closed without merging**